frontend/src/__tests__/cypress/cypress/tests/mocked/home/homeResources.cy.ts

```diff
@@ -11,21 +11,24 @@ describe('Home page Resources section', () => {
     homePage.initHomeIntercepts();
     homePage.visit();
   });
+
   it('should show the resources section', () => {
-    cy.findByTestId('landing-page-resources').scrollIntoView();
-    cy.findByTestId('resource-card-create-jupyter-notebook').should('be.visible');
+    const homeResourceSection = homePage.getHomeResourceSection();
+    homeResourceSection.find().scrollIntoView();
+    homeResourceSection.findJupyterCard().should('be.visible');
```

Comment:
Cards are dynamic based on the intercept of `GET /api/quickstarts`. Since the test is in control of which cards get displayed, the test should be passing in the card ID to assert.

```suggestion
    homeResourceSection.findCard('create-jupyter-notebook').should('be.visible');
```